blockmanager.go

```diff
@@ -2547,3 +2549,68 @@ func (b *blockManager) findPrevTestNetDifficulty(hList headerlist.Chain) (uint32
 	}
 	return lastBits, nil
 }
+
+// onBlockConnected queues a block notification that extends the current chain.
+func (b *blockManager) onBlockConnected(header wire.BlockHeader, height uint32) {
+	select {
+	case b.blockNtfnChan <- blockntfns.NewBlockConnected(header, height):
+	case <-b.quit:
+	}
+}
+
+// onBlockDisconnected queues a block notification that reorgs the current
+// chain.
+func (b *blockManager) onBlockDisconnected(headerDisconnected wire.BlockHeader,
+	heightDisconnected uint32, newChainTip wire.BlockHeader) {
+
+	select {
+	case b.blockNtfnChan <- blockntfns.NewBlockDisconnected(
+		headerDisconnected, heightDisconnected, newChainTip,
+	):
+	case <-b.quit:
+	}
+}
+
+// Notifications exposes a receive-only channel in which the latest block
+// notifications for the tip of the chain can be received.
+func (b *blockManager) Notifications() <-chan blockntfns.BlockNtfn {
```

Comment:
It doesn't, but it's currently necessary to avoid a wrapper struct in order to satisfy the interface.  Also, it's expected for only one client to read from it, which is the `blockntfns.SubscriptionManager`.